README.md

```diff
@@ -18,7 +18,9 @@
 
 ## Introduction
 
-An implementation of [libp2p](https://libp2p.io/) in Nim. Also provides a Nim wrapper of the [Libp2p Go daemon](https://github.com/libp2p/go-libp2p).
+An implementation of [libp2p](https://libp2p.io/) in Nim.
+
+Also optionally provides a Nim wrapper of the [Libp2p Go daemon](https://github.com/libp2p/go-libp2p).
 
 ## Project Status
 The current native Nim libp2p implementation support is experimental and shouldn't be relied on for production use. It is under active development and contributions are highly welcomed. :)
```

Comment:
Maybe this can be changed since we're releasing v1.

Reply:
By changing, I mean reworded :)

Reply:
Ah yeah :) indeed , let's not forget that